app/views/candidate_mailer/application_rejected/offers_made.erb

```diff
@@ -0,0 +1,40 @@
+Dear <%= @application.candidate_name %>,
+
+# Application decision
+
+<%= @application.provider_name %> has decided not to progress your teacher training application for <%= @application.course_name %> on this occasion. They gave the following feedback:
+
+<%= @application.rejection_reason %>
+
+Contact <%= @application.provider_name %> directly if you have any questions about this.
+
+# Make a decision about your <%= 'offer'.pluralize(@application.offers.count) %> by <%= @application.decline_by_default_at %>
+
+<% if @application.offers.count > 1  %>
+
+You’ve received the following offers:
+<% @application.offers.each do |offer| %>
+  - <%= "#{offer.course.name_and_code} at #{offer.provider.name}"%>
+<% end %>
+
+If you don’t reply within 10 working days, your applications will be withdrawn.
```

Comment:
The `10` should not be hard-coded – we should be using the `decline by default` duration out of the time limit config.

Reply:
Thanks